projects/pybind11.cmake

```diff
@@ -0,0 +1,6 @@
+superbuild_add_project(pybind11
```

Comment:
I had to duplicate this file from the official `pybind11.cmake` to cut the dependency to `python` which is installed externally.

Reply:
Ill investigate if i should make it optional in the common sb

Reply:
Basically, we should be able to enable `pybind11` without `python`, and add `python` as an optional dependency for `f3d`. Then `F3D_BINDINGS_PYTHON` should be enabled if `pybind11` AND `python` is enabled. What do you think?

Alternatively, enabling `python` should enable `pybind11` automatically, and not the other way around like in `common-superbuild`

Reply:
first one is indeed the best one

Reply:
https://github.com/f3d-app/f3d-superbuild/pull/84